FLEDGE.md

```diff
@@ -130,10 +131,10 @@ const myGroup = {
                    {renderUrl: gymShoes, sizeGroup; 'group2', ...},
                    {renderUrl: gymTrainers1, sizeGroup: 'size4', ...},
                    {renderUrl: gymTrainers2, sizeGroup: 'size4', ...}],
-  'adSizes': {'size1': {width: width1, height: height1},
-              'size2': {width: width2, height: height2},
-              'size3': {width: width3, height: height3},
-              'size4': {width: width4, height: height4}},
+  'adSizes': {'size1': {width: '100', height: '100'},
+              'size2': {width: '100px', height: '200px'},
+              'size3': {width: '200sw', height: '200sw'},
```

Comment:
From my other comment, `'200sw'` is not an appropriate use, correct?

Reply:
I was referring to this example specifying that the `height` value is relative to the screen _width_. Which isn't logical, but I suppose Chrome's position is that it's not responsiblle for illogical, but spec-valid configurations.

Reply:
Thanks for the clarification!  That makes sense.  I've switched them to only use sw for width, and sh for height.